### PR TITLE
Updating Readme

### PR DESCRIPTION
**Pull request by msmakouz**

### CODE REVIEW

**Comment by msmakouz** [comment]
@Sushisource https://github.com/temporalio/sdk-php-interceptors-opentelemetry/pull/1#pullrequestreview-1475565557

> Something that would be useful for our understanding and generally in the readme would be a diagram that shows what the spans look like (what spans exist, the parent/child/follows-from relationships, etc) for a normal flow where a client starts a workflow which executes some activities, for example.

Hello, which diagram would you like to see and where? Screenshot from Zipkin?

**Comment by Sushisource** [comment]
Yeah, that'd be perfect I think :+1:

File: README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ These interceptors allow you to capture and trace various actions and events, su
 sending signals to workflows, and executing workflow events. By integrating OpenTelemetry tracing, you gain visibility
 into the behavior and performance of your Temporal applications.
 
+<img width="1511" alt="otel" src="https://github.com/msmakouz/sdk-php-interceptors-opentelemetry/assets/67324318/96b3e7cf-a815-47bd-b09a-00214ea2381d">
```
Comment (Sushisource) [inline README.md]: Probably would be good to have this link be relative and within this repo

**Review by Sushisource — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (msmakouz) [reply README.md]: @Sushisource I'm sorry, didn't pay attention. I will update the image.

Comment (msmakouz) [reply README.md]: @Sushisource The image updated.

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**